akka-cluster-tools/src/multi-jvm/scala/akka/cluster/client/ClusterClientSpec.scala

```diff
@@ -141,12 +249,12 @@ class ClusterClientSpec extends MultiNodeSpec(ClusterClientSpec) with STMultiNod
 
       //#server
       runOn(host1) {
-        val serviceA = system.actorOf(Props[Service], "serviceA")
+        val serviceA = system.actorOf(Props[Service](), "serviceA")
```

Comment:
My IDE highlighted that - and I think it is correct - would you prefer that I restore it?